libdeno/binding.cc

```diff
@@ -558,7 +558,11 @@ v8::MaybeLocal<v8::Module> ResolveCallback(v8::Local<v8::Context> context,
 
   if (d->resolve_module_.IsEmpty()) {
     // Resolution Error.
-    isolate->ThrowException(v8_str("module resolution error"));
+    std::stringstream err_ss;
+    err_ss << "NotFound: Cannot resolve module \"" << specifier_c
```

Comment:
I don't like that this is faking an `ErrorKind::NotFound`...

Reply:
I just tried to simulation TS compiler error format. The problem is there is no real stack trace I could create. As mentioned above I could have used the two module request location calls to fake one (to show on which line of import that this error relates to) but this requires injecting something into the generated error JSON object — there is no API to directly modify the Exception object itself